### ISSUE TITLE
Being able to change language or subs from movie/TV menu using a gamepad

### ISSUE THREAD

**Issue by awsms** [jellyfin/jellyfin-media-player#283]
exactly the same as https://github.com/jellyfin/jellyfin-web/issues/3755

**Describe The Bug** Using Gamepad with TV layout enabled, I'm unable to interact with drop-down list to select language or subtitles. The field is highlighted but if I press A on the gamepad nothing happens. All other buttons (Play, Trailer, Back, etc) work fine.

**Steps To Reproduce**

1. Enable TV layout in jellyfin web
2. Enable gamepad support
3. Using gamepad, open a movie menu with more than one language and/or subtitles

**Expected Behavior** I should be able to select audio track or subtitles

**System (please complete the following information):**

* Platform: Windows 10
* Jellyfin Version: 10.8.1.0

**Comment by iwalton3** [comment]
If it is a jelyfin-web issue I doubt I can do much. But you may have some luck with rebinding the controls in the config files.

**Comment by iwalton3** [comment]
(This is unfortunately undocumented and under implemented.)

**Comment by awsms** [comment]
> If it is a jelyfin-web issue I doubt I can do much. But you may have some luck with rebinding the controls in the config files.

It's both a jellyfin-web issue and jellyfin-media-player, changing audio/subtitles tracks on the show/movie page won't work with a controller

**Comment by iwalton3** [comment]
Can you do it with the keyboard?

**Comment by awsms** [comment]
> Can you do it with the keyboard?

Yup, with pressing tab untill on the selector, then pressing spacebar. However that pops-up a different kind of selector than clicking on it with the mouse.

Mouse click:
![image](https://user-images.githubusercontent.com/48278661/179386660-193a26a8-7195-4e95-80f6-814a444b4e42.png)

Spacebar:
![image](https://user-images.githubusercontent.com/48278661/179386649-a4ab7055-5e64-4f7c-b0f9-3dd6f54726a2.png)

**Comment by iwalton3** [comment]
Hmm yeah that looks like it is using the native UI element which JF web can’t control with the controller implementation

**Comment by andrewrabert** [comment]
This Qt desktop client has been deprecated in favor of a rewrite: https://github.com/jellyfin/jellyfin-desktop
